### DISCUSSION TITLE
İki çocuklu adam olasılık sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#248] [Zeka Sorusu]
Bir adamın iki çocuğu varmış. Birisi kız olduğuna göre diğerinin erkek olma olasılığı nedir?

**Comment by Alperg123456** [comment]
1. çocuğun cinsiyeti 2. çocuğun cinsiyetini etkilemez o yüzden 2. çocuğun erkek olma ihtimali %50' dir

**Reply by sonsuzus** [reply]
Programını istiyorum.

**Comment by yukseel** [comment]
```
class Program
{
    static void Main(string[] args)
    {
        List<string> cinsiyetler = new List<string>
        {
            "Erkek",
            "Kız",
        };

        int toplamcinsiyet = cinsiyetler.Count;
        double olasilik = 1.0 / toplamcinsiyet*100;

        Console.WriteLine("Olasılık: %" + olasilik);

        Random random = new Random();
        int secilenCinsiyet = random.Next(toplamcinsiyet);
        string yazdir = cinsiyetler[secilenCinsiyet];

        Console.WriteLine("Cinsiyet " + yazdir);
    }
}``
```
İki cinsiyet olduğu için listedeki 2 cinsiyetten birinin gelme olasılığını hesaplayıp önce olasılığı sonra da gelen cinsiyeti %50 olasılığa göre rastgele çekip yazdıran kod

**Reply by sonsuzus** [reply]
Kodda birisinin kız olduğu tespiti nerede?

**Reply by yukseel** [reply]
kod içinde nasıl kullanabilirim ki? sadece yazdırmam yeterli mi yoksa listeden kız elemanını mı çekeyim
